decred_wallet/Features/Security/PinInputView.swift

```diff
@@ -56,13 +54,12 @@ class PinInputView: UIView {
     
     func drawPinLabel() {
         let pinDigitsCount = String(pin.count)
-        
-        let pinLabel = UILabel()
-        pinLabel.text = String(pinDigitsCount)
+        let pinLabel = UILabel(frame: self.bounds) // Set the label bounds to resolve any ambiguity
```

Comment:
If you pass in `frame` from the previous function, you can use it here.

Reply:
Done in commit 53544e6f739ec5e207ba40766aef7f13eabe85a8